contracts/Unibright.sol

```diff
@@ -24,22 +24,12 @@ import "@openzeppelin/contracts/access/Ownable.sol";
  * function.
  */
 contract UBTSplitter is Context, Ownable {
-    event PayeeAdded(
```

Comment:
payeeadded and payeeupdated are same events, it is easier to only throw payeeupdated